File: src/main/java/com/wgzhao/addax/admin/repository/NotificationRepo.java
package com.wgzhao.addax.admin.repository;

import com.wgzhao.addax.admin.model.Notification;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Date;
import java.util.List;

/**
 * 数据中心消息提醒总表
 *
 * @author 
 */
public interface NotificationRepo
        extends JpaRepository<Notification, String> {
    List<Notification> findDistinctBydwCltDateAfter(Date td);
}

File: src/main/kotlin/com/wgzhao/addax/admin/repository/NotificationRepo.kt
package com.wgzhao.addax.admin.repository

import com.wgzhao.addax.admin.model.Notification
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

/**
 * 数据中心消息提醒总表
 *
 * @author
 */
interface NotificationRepo

    : JpaRepository<Notification?, String?> {
    fun findDistinctByCreateAtAfter(td: Date?): MutableList<Notification?>?

    fun findAllByImOrderByCreateAtAsc(y: String?): MutableList<Notification?>?
}